src/components/index/IndexComponent.tsx

```diff
@@ -99,7 +103,7 @@ const Index = ({
   }, [descendingRequests]);
 
   function filterDescendingRequests(filter: Filter) {
-    let fr: RequestIndex[] = [];
+    let fr: Requests = [];
```

Comment:
the request type has been unified for the index page and request details, so now its just called `Requests`